### ISSUE TITLE
Unable to change SymmetryThreshold of Imager

### ISSUE THREAD

**Issue by saikns** [lovepark/o3d3xx-ros#33]
Following a recent discussion with IFM reps, I was trying to change the symmetry threshold of the imager to eliminate some over saturated pixels due to motion. I tried changing symmetry threshold through the config node but it's not reflected in the imager. ( json dump just returns a 0 value after the config change)

`"Imager":
{
  "SymmetryThreshold": "0"
}`

I dug deeper & found this commnet in libo3d3xx/modules/camera/src/libo3d3xx_camera/imager_config.cpp
`//! @todo: This data type may change. It is not clear that this is even
//! implemented in the camera yet. I believe it has to do with flagging a
//! pixel as bad due to motion artifacts across exposure times ... but it
//! does not appear to be documented.
int SymmetryThreshold() const noexcept;`

I was wondering if it's not an accessible parameter or is it not implemented?
@tpanzarella, @garrettplace any comments?

**Comment by tpanzarella** [comment]
Use the `o3d3xx-ifm-export` application to create a dump compatible with Vision Assistant. Then edit that file (Zip'd JSON), then reimport it with `o3d3xx-ifm-import`. Something like (assuming the application you want to edit is at Index 1):

```
$ o3d3xx-ifm-export -i 1 -a -f /tmp/app.json.zip
``` 

Now edit the application using standard OS tools. Then:

```
$ o3d3xx-ifm-import -a -f /tmp/app.json.zip
```

**Comment by saikns** [comment]
I am using ros service calls since I am having trouble using command-line-utils, all the commands return 
`Failed to * XMLRPC call timed out` 
Any idea why that could happen or is there a way to do it through ros?

**Comment by tpanzarella** [comment]
I don't believe we expose that feature via a ROS service. You could, of course, write code directly to the API in a ROS node, but I think that would be a waste of your time.

Per the timeouts, typically, it is due to running the camera on a non-default IP address. The default is 192.168.0.69. If no additional information is given, that is assumed. You can specify the IP address in one of two ways: 1) using the `--ip=<ip address>` command line parameter, or 2) setting the `O3D3XX_IP` environment variable which the library will respect. FWIW, all cmd line tools support a `--help` option letting you know their accepted parameters.
